src/components/Projects.tsx

```diff
@@ -1,7 +1,149 @@
+import statue from "./assets/project-page/statue.png";
+import dragon from "./assets/project-page/dragon.png";
+import knife from "./assets/project-page/knife.png";
+import mask from "./assets/project-page/mask.png";
+import spider from "./assets/project-page/spider.png";
+import magnify from "./assets/project-page/magnify.png";
+import Background from "./assets/home-page/background.jpg";
+import {
+  color,
+  space,
+  typography,
+  TypographyProps,
+  ColorProps,
+  SpaceProps,
+} from "styled-system";
+
+type HeadingProps = SpaceProps & ColorProps & TypographyProps;
+
+const Header = styled.div<HeadingProps>`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+`;
+
+const Title = styled.h1<HeadingProps>`
+  ${space}
+  ${color}
+  ${typography}
+  font-family: SangBleu OG Serif Light Regular;
+`;
+
+const IconContainer = styled.div`
+  width: 200px;
+  height: 120px;
+`;
+
+const Icon = styled.a`
+  a:hover {
+    position: relative;
+    width: 500px;
+    height: auto;
+  }
+`;
 
 const Projects = () => {
-  return <div>Projects</div>;
+  return (
+    <div
+      className="bg_image"
+      style={{
+        backgroundImage: "url(" + Background + ")",
+        backgroundSize: "cover",
+        height: "140vh",
+      }}
+    >
+      <Header>
+        <Link to="/" style={{ textDecoration: "none" }}>
+          <Title p={7} color="athensGray" fontSize={5}>
+            OXYMORE
+          </Title>
+        </Link>
+      </Header>
+      <IconContainer>
+        <Icon>
+          <img
+            src={stairs}
+            style={{
+              position: "relative",
+              top: 100,
+              left: 100,
+            }}
+          ></img>
+        </Icon>
+      </IconContainer>
+      <IconContainer>
+        <Icon>
+          <img
+            src={shell}
+            style={{ position: "relative", top: 240, left: 100 }}
+          ></img>
+        </Icon>
+      </IconContainer>
+      <IconContainer>
+        <Icon>
+          <img
+            src={eye}
+            style={{ position: "relative", bottom: 240, left: 420 }}
+          ></img>
+        </Icon>
+      </IconContainer>
+      <IconContainer>
+        <Icon>
+          <img
+            src={statue}
+            style={{ position: "relative", top: 54, left: 420 }}
+          ></img>
+        </Icon>
+      </IconContainer>
+      <IconContainer>
+        <Icon>
+          <img
+            src={dragon}
+            style={{ position: "relative", bottom: 260, left: 720 }}
+          ></img>
+        </Icon>
+      </IconContainer>
+      <IconContainer>
+        <Icon>
+          <img
+            src={knife}
+            style={{ position: "relative", bottom: 40, left: 680 }}
+          ></img>
+        </Icon>
+      </IconContainer>
+      <IconContainer>
+        <Icon>
+          <img
+            src={mask}
+            style={{ position: "relative", bottom: 720, left: 1020 }}
```

Comment:
See above comment about using styled-system.
Also these "bottom" and "left" values will need to be responsive

Reply:
etc etc for the other IconImg's aswell!

Reply:
Again, sorry. This was my pre-responsive/ styled-system brain at work. Will push up a fresh PR with the changes so you can have a go with the different screen sizes. 

**Question** - On mobile view 320 x 683, you can see that the icons looks a little off kilter compared to the next screen-width up, 375 x 683.

<img width="1440" alt="Screenshot 2020-07-03 at 14 12 25" src="https://user-images.githubusercontent.com/53219789/86468279-62ffbe80-bd37-11ea-9fdb-d001342da8d1.png">
<img width="1440" alt="Screenshot 2020-07-03 at 14 12 32" src="https://user-images.githubusercontent.com/53219789/86468286-67c47280-bd37-11ea-8787-8921c497c063.png">

To solve I created a container `div` named `IconsContainer` to nest the `Icons` in their respective place and center the container itself taking inspiration from this logic that you posted above: 

> Since this image is centered, could we have it statically positioned (ie. default, ie. remove position="absolute"), and remove the top=...
... and add alignItems="center" to <Background>

It didn't quite work, any tips?

Reply:
The centered container sounds like a great shout.